master/internal/webhooks/postgres_webhook.go

```diff
@@ -122,6 +123,26 @@ func (l *WebhookManager) removeTriggers(triggers []*Trigger) error {
 	return nil
 }
 
+func (l *WebhookManager) editTriggers(ts []*Trigger) error {
+	l.mu.Lock()
+	defer l.mu.Unlock()
+
+	for _, t := range ts {
+		if t.TriggerType != TriggerTypeTaskLog {
+			return nil
```

Comment:
is it expected that you receive requests to edit non-TaskLog triggers? Also do we want to exit the function as soon as we encounter the first non-Task log type trigger?

Reply:
> is it expected that you receive requests to edit non-TaskLog triggers? 
Yes. I'm skipping it because only TaskLog triggers are stored in cache. https://github.com/determined-ai/determined/pull/9890/files/c08e8b902b779999dee41e3d7b287f67453d8750#diff-3ea86afd178c30910e3fdbc3f40b950a61d566b1e70bc4d932bbba3e693c2802R71-R78

>  Also do we want to exit the function as soon as we encounter the first non-Task log type trigger?
Good catch! I shouldn't exit the function.